tests/fXmlRpc/Parser/AbstractParserTest.php

```diff
@@ -697,4 +697,17 @@ public function testEmptyStructMember()
         $this->assertSame(array('FIRST' => ''), $this->parser->parse($string, $isFault));
         $this->assertFalse($isFault);
     }
+
+    public function testThrowExceptionWhenIsString()
```

Comment:
Can you add a testcase that tests that the validation doesn’t happen if the flag is not set?